AssettoServer/Network/Tcp/ACTcpClient.cs

```diff
@@ -847,24 +846,27 @@ private async Task SendFirstUpdateAsync()
                     }
                 });
             }
-            
+
+            _entryCarManager.BroadcastPacket(CreateLapCompletedPacket(0xFF, 0, 0));
```

Comment:
as far as I can see this packet isn't broadcasted in the original acserver. Any specific reason you changed this?

Reply:
Ah, not sure about to og code, but I put this in so other players' leaderboards are updated whenever a new player joins. Otherwise, the leaderboard only updates for the joining user and the other users won't see the updated lap info unless they re-join or another LapComplete is broadcasted by someone completing a lap.